clvm/operators.py

```diff
@@ -53,15 +62,13 @@ def f(args):
                 return self.unknown_op_handler(op, args)
         return f(arguments)
 
-    def set_unknown_op_handler(self, callback: Callable[[bytes, CLVMObject], Tuple[int, CLVMObject]]):
-        self.unknown_op_handler = callback
-
     def unknown_op_raise(self, op: bytes, arguments: CLVMObject):
         raise EvalError("unimplemented operator", arguments.to(op))
 
+QUOTE_ATOM = KEYWORD_TO_ATOM["q"]
+APPLY_ATOM = KEYWORD_TO_ATOM["a"]
 
 OPERATOR_LOOKUP = OperatorDict(
-    operators_for_module(KEYWORD_TO_ATOM, core_ops, OP_REWRITE)
+    operators_for_module(KEYWORD_TO_ATOM, core_ops, OP_REWRITE), quote=QUOTE_ATOM, apply=APPLY_ATOM
```

Comment:
I have a feeling you may be able to use `KEYWORD_TO_ATOM["q"]` and `KEYWORD_TO_ATOM["a"]` directly here, and scrap the `QUOTE_ATOM` and `APPLY_ATOM` globals

Reply:
Yes - I'll remove these entirely, now that I have the ability to either "inherit" them from another `OperatorDict`, or set them at construction time.